indexer/search_string_utils.cpp

```diff
@@ -130,6 +130,10 @@ UniString NormalizeAndSimplifyString(std::string_view s)
       c = 'a';
       uniString.insert(uniString.begin() + (i++) + 1, 'e');
       break;
+    case 0x2018:  // ‘
+    case 0x2019:  // ’
+      c = '\'';
```

Comment:
Can we reuse the same constant if it's also used somewhere in the search engine, for clarity, why is it replaced here?